bundle/src/main/java/com/adobe/acs/commons/replication/dispatcher/DispatcherFlushFilter.java

```diff
@@ -130,13 +130,13 @@ private boolean isEnabled(final Agent agent) {
     }
 
     /**
-     * Checks if the agent has a "flush" serialization type.
+     * Checks if the agent's serialization type starts with "flush" .
      *
      * @param agent Agent to check
-     * @return true if the Agent's serialization type is "flush"
+     * @return true if the Agent's serialization type starts with "flush"
      */
     private boolean isFlushingAgent(final Agent agent) {
-        return StringUtils.equals(SERIALIZATION_TYPE, agent.getConfiguration().getSerializationType());
+        return StringUtils.startsWith(agent.getConfiguration().getSerializationType(), SERIALIZATION_TYPE);
```

Comment:
@davidjgonzalez 
I'd like your opinion on this change to existing code.  Instead of having the agent be exactly "flush", the check changed to be starting with "flush".  That'll be backwards compatible.  Will there be collisions?  More than 1 agent trying to do the activation if configured incorrectly?
Thoughts?